test/common.py

```diff
@@ -381,15 +381,35 @@ def metafunc(self, wasmfs, *args, **kwargs):
     if wasmfs:
       self.set_setting('WASMFS')
       self.emcc_args.append('-DWASMFS')
-      f(self, *args, **kwargs)
-    else:
-      f(self, *args, **kwargs)
+    f(self, *args, **kwargs)
 
   parameterize(metafunc, {'': (False,),
                           'wasmfs': (True,)})
   return metafunc
 
 
+def also_with_wasmfs_and_rawfs(func):
+  assert callable(func)
+
+  @wraps(func)
+  def metafunc(self, rawfs, wasmfs, *args, **kwargs):
+    if DEBUG:
+      print('parameterize:rawfs=%d,wasmfs=%d' % (rawfs, wasmfs))
+    if rawfs:
+      self.require_node()
+      self.emcc_args += ['-DNODERAWFS']
+      self.set_setting('NODERAWFS')
+    elif wasmfs:
+      self.set_setting('WASMFS')
+      self.emcc_args.append('-DWASMFS')
+    func(self, *args, **kwargs)
+
+  parameterize(metafunc, {'': (False, False),
+                          'rawfs': (True, False),
+                          'wasm2js': (False, True,)})
```

Comment:
Does WasmFS not support NODERAWFS? If not then it could be worth a TODO here to remove this decorator when that is fixed, at which point we could do `@also_with_wasmfs` on top of `@also_with_rawfs`.

Reply:
Oh yes it looks like it does.  I'll add another config for that here!